src/controllers/rooms.controller.js

```diff
@@ -0,0 +1,73 @@
+const { Room } = require('../models/room.model');
+const roomsService = require('../services/rooms.service');
+const { ApiError } = require('../utils/api.error');
+
+const getAll = async (req, res) => {
+  const rooms = await roomsService.getAll();
+
+  res.send(rooms);
+};
+
+const create = async (req, res) => {
+  const { name, userId } = req.body;
+
+  if (!name || !userId) {
+    throw ApiError.badRequest('Not all data is provided!');
+  }
+
+  const possibleRoom = await Room.findOne({ where: { name } });
+
+  if (possibleRoom) {
+    throw ApiError.badRequest('This room already exists');
+  }
+
+  const newRoom = await roomsService.create({ name, userId });
+
+  res.send(newRoom);
+};
+
+const remove = async (req, res) => {
+  const { id } = req.params;
+
+  if (!id) {
+    throw ApiError.notFound();
```

Comment:
Consider using a more descriptive error message for when the room ID is not provided, such as 'Room ID is required.'